### PR TITLE
Add support for serialized rollback

### LINKED ISSUE #329: Tests using live_server fixture removing data from data migrations
I've created a simple test case to reproduce this behavior https://github.com/ekiro/case_pytest/blob/master/app/tests.py which fails after second test using live_server fixture. 
MyModel objects are created in migration, using RunPython. It looks like after any test with live_server, every row from the database is truncated.

### PR DESCRIPTION
**Pull request by blueyed**
Fixes #329.

TODO:
- [ ] add test(s)

This conflicts (in the sense of touching the same code) with https://github.com/pytest-dev/pytest-django/pull/308.

Continuation/rebase of #331.

### CODE REVIEW

**Comment by pelme** [comment]
As per my comment in PR #308 (https://github.com/pytest-dev/pytest-django/pull/308#issuecomment-230173857) I think the fixture in this PR should be named to `django_db_serialized_rollback`.

File: pytest_django/fixtures.py [hunk 1]
```diff
@@ -130,7 +134,23 @@ def transactional_db(request, _django_db_setup, _django_cursor_wrapper):
...
+                              request=request,
+                              _django_cursor_wrapper=_django_cursor_wrapper)
+
+
+@pytest.fixture(scope='function')
+def serialized_rollback(request):
+    """Enable serialized rollback after transaction test cases
+
+    This fixture only has an effect when the ``transactional_db``
+    fixture is active, which happen as a side-effect of requesting
```
Comment (blueyed) [inline pytest_django/fixtures.py]: s/happen/happens/

**Comment by atodorov** [comment]
ping. what's the status of this PR? Is there a newer version? What do you need help with? I kind of need this fixed ASAP and I can help clean up the rest of the code review if you can merge quickly.

**Comment by blueyed** [comment]
@atodorov 
Please pick it up and create a new PR then.

**Comment by bluetech** [comment]
A PR which will replace this PR is in #970. I intend to merge and release it in a few days. Comments and tests are welcome.

**Pull request closed without merging**